root/static/scripts/release-editor/fields.js

```diff
@@ -67,9 +67,13 @@ class Track {
     this.name.original = data.name;
     this.name.subscribe(this.nameChanged, this);
 
-    this.hasFeatOnOrigTitle = featRegex.test(
-      data.name.replace(bracketRegex, ' '),
-    );
+    /*
+     * Check that the track has an ID to avoid treating a newly-added title
+     * as original when seeding a new release (see MBS-13920).
+     */
+    this.hasFeatOnOrigTitle =
+      this.id != null &&
```

Comment:
Is this safe for checking that the track already existed?

Reply:
Thanks, I've updated this statement to use `isDatabaseRowId` since it feels a bit safer.